src/pyclaw/sharpclaw/solver.py

```diff
@@ -174,13 +177,21 @@ def __init__(self,riemann_solver=None,claw_package=None):
         self._method = None
         self._registers = None
         self.dq_dt = None
-        self.dt_old = None        
+        self.dwdq_dt = None
```

Comment:
I would call this something like `dq_dt_downwind`; the current name seems to imply differentiation with respect to w.